packages/forms-web-app/__tests__/setupTests.js

```diff
@@ -4,14 +4,21 @@ const fetchMock = require('jest-fetch-mock');
 
 fetchMock.enableMocks();
 
+const clearAllObjValues = (translations) =>
+	Object.keys(translations).forEach((key) => {
+		if (typeof translations[key] === 'object') {
+			clearAllObjValues(translations[key]);
+		} else translations[key] = '';
+	});
+
 function toHaveSameKeys(a, b) {
-	const aKeys = Object.keys(a).sort();
-	const bKeys = Object.keys(b).sort();
+	clearAllObjValues(a);
+	clearAllObjValues(b);
```

Comment:
This is a nice solution, but it might have unintended side effects on other tests that run after the `.toHaveSameKeys` assertion because the underlying object is being mutated.

You could instead have `clearAllObjValues` construct a new object. Something like:

```javascript
const clearAllObjValues = (translations) =>
  Object.keys(translations).reduce((acc, key) => {
    if (typeof translations[key] === 'object') {
      acc[key] = clearAllObjectValues(translations[key]);
    } else {
      acc[key] = '';
    }

    return acc;
  })
```